upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template

```diff
@@ -167,7 +167,7 @@ spec:
           beta.kubernetes.io/os: linux
       containers:
       - name: coredns
-        image: k8s.gcr.io/coredns:1.6.6
+        image: k8s.gcr.io/coredns:1.6.7
```

Comment:
The image for 1.6.7 isn't available in k8s.gcr.io...

It makes me wonder how https://github.com/kubernetes/kops/pull/8452 and other cherry-pick PRs have passed tests.

Reply:
the presubmit e2e tests still use kubedns since it is the kops default. Maybe this is a good time to bring attention to your #7919 :)

Reply:
Aah! that makes sense.

Reply:
What's the problem here to use `coredns/coredns:1.6.7` instead?

Reply:
I believe the kubernetes project moved from coredns/coredns to k8s.gcr.io/coredns so as to try to make sure that critical dependencies have a coherent "supply chain" - in particular the work going on in wg-k8s-infra.  It's very much a work-in-progress though.

The good news:  k8s.gcr.io/coredns:1.6.7 is now available :-)

Reply:
@justinsb I think the CoreDNS 1.6.7 version was added under #8602.
I don't see it cherry-picked to 1.15 yet or any release note for it.